src/sentry/integrations/github_enterprise/webhook.py

```diff
@@ -127,7 +128,7 @@ class GitHubEnterpriseWebhookBase(Endpoint):
     authentication_classes = ()
     permission_classes = ()
 
-    _handlers: dict[str, Callable[[], Callable[[Any], Any]]] = {}
+    _handlers: dict[str, type[InstallationEventWebhook] | type[Webhook]] = {}
```

Comment:
What is this doing? Just correcting the type?

Reply:
fixing the type so mypy knows `event_handler.event_type` is the correct type https://github.com/getsentry/sentry/pull/81594/files#diff-32aff156b2d2b2b2a975d63b960b8dc6aafdc6f01e9f535be23de0cb8988f6b1R302